tests/does-it-work/src/main.rs

```diff
@@ -81,15 +81,19 @@ mod tests {
             mangled_function
         );
 
-        let function = parts.next().expect("Expected a function name");
-        assert!(
-            function.contains("[does_it_work::run_test::"),
-            "Function name appears incorrect: {}",
-            function
-        );
-
-        let probe = parts.next().expect("Expected a probe name");
-        assert_eq!(probe, "work", "Probe name appears incorrect: {}", probe);
+        // These parts are currently specific to Mac OS X:
```

Comment:
I'd really just prefer remove this block, actually. I've realized it's testing a platform-specific flavor of the output, which seems to demangle the function name automatically. The test adds nothing beyond the surrounding bits, and I think the conditional-compilation will raise more questions than it's worth.

Reply:
Sold!